index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Poppins:ital,wght@0,200;0,500;0,600;0,800;1,100&display=swap" rel="stylesheet">
+    <title>runner world cup</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+</head>
+<body>
+    <span class="upperhead">
+        <div>
+            <img src="./images/Rectangle.png" alt="">
+            <img src="./images/twitter.png" alt="">
+            <a href="#">English</a>
+            <a href="#">MyPage</a>
+            <a href="#">Logout</a>
+        </div>
+    </span>
+<header>
+    <nav>
+        <div class="burger">
+            <img src="./images/mobile_menu 1.png" alt="" />
+        </div>
+            <div class="header-menu">
+            <span class="close">X</span>
+            <a class="links" id="about" href="./MobileSecondPage.html">About</a>
+            <a class="links" id="myProgram" href="#myProgram">Program</a>
+            <a class="links" id="join" href="#join">Join</a>
+            <a class="links" id="sponsor" href="#sponsor">Sponsor</a>
+            <a class="links" id="news" href="#news">news</a>
+            <a href="#" id="DeskIcon">Worldcup</a>
+        </div>
+    </nav>
+    <div class="logo">
+    <a href="./index.html"><img src="./images/Screenshot 2023-04-new .jpg" alt="world-cup"></a>
+    </div>
+</header>
+<section class="headline">
+    <div class="first-headline">
+        <P>"Hello! sharing world"</P>
+        <h1>RUNNING WORLD CUP 2015</h1>
+    </div>
+    <p class="main-content">This is your opportunity to represent your country. It is free to enter,taking part in the 2015 Vitality Running World Cup.</p>
+    <div class="second-headline">
+        <h2>2021 Tournament Highlights</h2>
+        <p>We can't thank our VRWC Runner enough for taking part in the 2015 Vitality Running World Cup.</p>
+    </div>
+</section>
```

Comment:
This section looks good on mobile 📱 but it lacks alignment on the Desktop Version, the texts should be aligned on the right, take a look at the differences in the layout 👇🏽 

## Behance

![Screenshot 2023-04-18 at 22 25 59](https://user-images.githubusercontent.com/45739963/232950557-ab1da11c-4a44-4932-96d0-dbe4387d8675.png)


## Your design

![Screenshot 2023-04-18 at 22 26 52](https://user-images.githubusercontent.com/45739963/232950698-53e7a807-97c0-4b4b-a2ec-85b79ca76b21.png)

Also,  make sure to have the text description in the box with the white border, increase the text sizes and you should be good to go 😉